src/components/molecules/faq.jsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const FaqContainer = styled.div`
```

Comment:
Put styles in separate styles.jsx

Reply:
done

Reply:
@Lucif3r-in @Shurtu-gal now I have to add an "+" icon each question , should I do that ri8 now?